docker-test.hcl

```diff
@@ -0,0 +1,83 @@
+param "test-case" {}
+
+param "converge-bin-dir" {}
+
+param "image-name" {
+  default = "ubuntu"
+}
+
+param "image-tag" {
+  default = "latest"
+}
+
+docker.image "converge" {
+  name = "{{param `image-name`}}"
+  tag  = "{{param `image-tag`}}"
+}
+
+task.query "rpc-token" {
+  query = "uuid"
+}
+
+docker.container "converge" {
+  name  = "converge-test"
+  image = "{{lookup `docker.image.converge.name`}}:{{lookup `docker.image.converge.tag`}}"
+
+  expose = ["4774"]
+
+  volumes = [
+    "{{param `converge-bin-dir`}}:/converge",
+    "{{env `PWD`}}/samples:/samples",
+    "{{env `PWD`}}/examples:/examples",
+  ]
+
+  entrypoint = ["/converge/converge"]
+  command    = ["server", "--rpc-token", "{{lookup `task.query.rpc-token.status.stdout`}}"]
```

Comment:
To be clearer: I would have expected `{{lookup "task.query.rpc-token.stdout" }}` since it only has one field. `task.query.rpc-token.query.stdout` at most.